build.sbt

```diff
@@ -113,11 +120,12 @@ val commonSettings = Seq(
     "akka.pattern" // for some reason Scaladoc creates this
   ),
   // show full stack traces and test case durations
-  testOptions += Tests.Argument("-oDF"),
+  testOptions += Tests.Argument(TestFrameworks.ScalaTest, "-oDF"),
   // -a Show stack traces and exception class name for AssertionErrors.
   // -v Log "test run started" / "test started" / "test run finished" events on log level "info" instead of "debug".
   // -q Suppress stdout for successful tests.
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-a", "-v", "-q"),
+  testOptions += Tests.Argument(jupiterTestFramework, "-a", "-v", "-q"),
```

Comment:
It is interesting that with this change these options are still used when running JUnit 4 tests.

Reply:
The JUnit 4 tests are now run by Jupiter.

Reply:
That makes sense. Then we should be able to get rid of the `"com.novocode" % "junit-interface"` dependency.